packages/office-addin-usage-data/src/usageData.ts

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT license.
 
 import * as appInsights from "applicationinsights";
+import * as appInsightsWeb from '@microsoft/applicationinsights-web';
```

Comment:
Did you test the new web methods in taskpane code (e.g. https://github.com/OfficeDev/Office-Addin-TaskPane/blob/master/src/taskpane/excel.ts).  We need to ensure that if the taskpane code calls the web methods that the project still builds.  Build fails when calling the non-web methods

Reply:
We should add tests to test/test.ts for the new web functionality

Reply:
I just tried importing this into Office-Addin-Taskpane and because usageData.ts also imports applicationinsights(the non-web version) build fails. I think this suggests that maybe we should have an entirely different package for web

Reply:
I just did some local simplifying of the new web package and am able to build. Basically, in order to be able to import to the taskpane and build, all references to fs and applicationinsights need to be removed.  We should discuss because I think the new package doesn't need to support all the cli commands that office-addin-usage-data supports. I confirmed with my local changes that the taskpane project successfully builds and reports usage data when the taskpane app is used